spa/src/pages/ConfigSteps/index.tsx

```diff
@@ -167,6 +165,7 @@ const ConfigSteps = () => {
 		}
 	}, []);
 
+	/*eslint-disable react-hooks/exhaustive-deps*/
```

Comment:
Do we need this disabling comment anymore?

Reply:
haha no, must be from previous